src/condor_utils/file_transfer.cpp

```diff
@@ -3497,6 +3505,9 @@ FileTransfer::DoDownload( filesize_t *total_bytes_ptr, ReliSock *s)
 				}
 				try_again = false;
 				formatstr(error_buf, "%s", errstack.getFullText().c_str());
+				if (result == TransferPluginResult::ExecFailed) {
+					try_again = true; // not the job's fault
+				}
```

Comment:
This looks silly but probably isn't worth checking a change to something like the following.
```suggestion
				}
				formatstr(error_buf, "%s", errstack.getFullText().c_str());
				try_again = (result == TransferPluginResult::ExecFailed;
```

Reply:
FWIW, my thought was that something else is going to in this block in the future.